algolia/algolia-search-bundle/2.2/config/packages/algolia.yaml

```diff
@@ -1,3 +0,0 @@
-algolia:
```

Comment:
No need to remove the old recipe

Reply:
I think it's important to remove it, people should use it anymore. I believe this is why the tests are failing. I'm not sure how to fix this 🤔

Reply:
If I really want to install 2.2 for some reason, shouldn't I be able to use the old recipe?